lib/aweplug/helpers/searchisko.rb

```diff
@@ -165,7 +165,13 @@ def post path, params = {}
         resp = @faraday.post do |req|
           req.url "/v1/rest/" + path
           req.headers['Content-Type'] = 'application/json'
-          req.body = params
+          if params.is_a? Hash
+            ENV["SEARCHISKO_DATA_ORIGIN"] ||= "build from #{ENV['USERNAME'] || ENV['USER']}"
```

Comment:
One of those is populated by the OS. I think USER is for Windows and USERNAME is *nix.